src/InverseComplementaryCumulativeDistribution.cpp

```diff
@@ -24,7 +24,7 @@ double InverseComplementaryCumulativeDistribution(double q)
     double D_3 = 0.001308;
```

Comment:
My really old copy of A&S and ITU-R P.1057 have C_0 set to 2.515517 (last digit is different).

Reply:
correct.  must have been a typo on my part.